database/migrations/2020_04_13_150500_add_last_error_fields_to_bgp_peers.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddLastErrorFieldsToBgpPeers extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('bgpPeers', function (Blueprint $table) {
+            $table->integer('bgpPeerLastErrorCode')->nullable()->after('bgpPeerAdminStatus');
+            $table->integer('bgpPeerLastErrorSubCode')->nullable()->after('bgpPeerLastErrorCode');
+            $table->string('bgpPeerLastErrorText', 254)->nullable()->after('bgpPeerLastErrorSubCode');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     *
+     *
+     *
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('bgpPeers', function (Blueprint $table) {
+            $table->dropColumn('bgpPeerLastErrorCode');
```

Comment:
This should be one dropColumn() with an array of all columns to drop.

Reply:
Done